tests/project/test_project.py

```diff
@@ -6,11 +6,39 @@
 from os.path import dirname, join, abspath
 from os import remove
 from pyiron_base.project.generic import Project
+from pyiron_gui import activate_gui
 from pyiron_gui.project.project_browser import ProjectBrowser
 from tests.toy_job_run import ToyJob
 import ipywidgets as widgets
 
 
+class TestActivateGUI(unittest.TestCase):
```

Comment:
Can now import and inherit from `pyiron_base._tests.TestWithProject` and reduce lines in setup and teardown

Reply:
Something goes really wrong with it, though... the `__file__` is replaced with the pyiron_base path and not with the path of the  test wich has to be run.

Reply:
The guys noticed this and I [patched it](https://github.com/pyiron/pyiron_base/pull/271); update base master and try again

Reply:
That's the reason it did not pop up in my test suite - I have an up to date base. To have the test to pass here, however, we need the fix to be in the conda package. I'll just wait until then since this is not urgent :)

Reply:
Indeed, I would like https://github.com/pyiron/pyiron_base/pull/278 to be also in, such that I can remove the `tearDown` pass.

Reply:
Sounds good! Sorry for the inconvenience due to my original mistake :shame:

Reply:
😄 I could just not use the new functionality for now. However, since I like it, I am happy to wait a bit.